### PR TITLE
Feature/display cards

### LINKED ISSUE #8: 2. USER STORY: As a user when i choose a state from the drop down menu, and submit, I am directed to a new screen where a list cards with a common bird name and a scientific bard name are visible and scrollable.
- [x] Can select one state from the drop down menu
- [x] After selecting a state and clicking the submit button, the user is taken to a new page with a list of birds seen in that state
- [x] The URL will update to reflect this change
- [x] Each bird card will have the bird's common and scientific name
- [ ] The navigation bar will be visible on the page
- [ ] The user can scroll down to see all bird cards

### PR DESCRIPTION
**Pull request by LKessell**
# Pull Request Template
## Description

Display fetched data as bird cards after submitting a state
Change URL using router after state choice

## Type of change
- [ ] Bug fix 
- [x] New feature
- [ ] Refactoring

## Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] My changes generate no new errors/warnings

### CODE REVIEW

File: src/components/StatePicker/StatePicker.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { getBirds } from '../../apiCalls';
```
Comment (sydnerd) [inline src/components/StatePicker/StatePicker.js]: It looks like we don't need this line because we don't use "getBirds" anymore in the code

**Comment by sydnerd** [comment]
It might be nice to have a loading indicator while the bird cards are loading. I found that it takes a while to load.

File: src/components/StatePicker/StatePicker.js (hunk 1 above)
Comment (LKessell) [reply src/components/StatePicker/StatePicker.js]: Good catch! I'll remove this 👍

**Comment by LKessell** [comment]
I love that idea, we should implement a loading screen for sure! Let's make a separate branch!

**Pull request merged**